app/views/downloads/_download_chapter.html.erb

```diff
@@ -4,36 +4,36 @@
   <h2 class="heading"><%= @chapter.chapter_title.html_safe %></h2>
   <% if (!@chapter.pseuds.blank? && (@chapter.pseuds != @work.pseuds) && (!@work.anonymous?)) %>
     <%# only display byline if different from the main byline %>
-    <p class="byline"><%= ts('Chapter by') %> <%= byline(@chapter, only_path: false) %></p>
+    <p class="byline"><%= t(".chapter_byline_html", byline_names_link: byline(@chapter, only_path: false)) %></p>
   <% end %>
 
   <% unless @chapter.summary.blank? %>
-    <p><%= ts('Chapter Summary') %></p>
-    <blockquote class="userstuff"><%=raw sanitize_field(@chapter, :summary) %></blockquote>
+    <p><%= t(".chapter_summary") %></p>
+    <blockquote class="userstuff"><%= raw sanitize_field(@chapter, :summary) %></blockquote>
   <% end %>
 
   <% unless @chapter.notes.blank? && @chapter.endnotes.blank? %>
-    <p><%= ts('Chapter Notes') %></p>
+    <p><%= t(".chapter_notes") %></p>
     <% unless @chapter.notes.blank? %>
-      <blockquote class="userstuff"><%=raw sanitize_field(@chapter, :notes) %></blockquote>
+      <blockquote class="userstuff"><%= raw sanitize_field(@chapter, :notes) %></blockquote>
     <% end %>
     <% unless @chapter.endnotes.blank? %>
       <div class="endnote-link">
-        <%= ts("See the end of the chapter for") %> <% unless @chapter.notes.blank? %><%= ts("more") %><% end %> <%= link_to ts("notes"), "#endnotes#{@chapter.position}" %>
+        <% t(".see_chapter_end_notes_html", chapter_end_notes_link: link_to(@chapter.notes.present? ? t(".more_notes_link") : t(".notes_link"), "#endnotes##{@chapter.position}")) %>
```

Comment:
The additional # in `#endnotes##{@chapter.position}` should be removed, it's breaking the anchor.